functions/utils.js

```diff
@@ -136,20 +137,29 @@ export function getNbCells(title, zipRes, argparser, template) {
     'Please, run the cell below to execute your code.'
   ]
 
-  const common_nb_commands = [
+  let common_nb_commands = [
     `!wget ${zipRes}\n`,
     `!unzip ${template}.zip\n`,
     '!pip install -r requirements.txt'
   ]
 
-  const execution_nb_commands = [
+  let execution_nb_commands = [
     `!python main.py ${
       argparser === 'hydra'
         ? '#--config-dir=/content/ --config-name=config.yaml'
         : 'config.yaml'
     }`
   ]
 
+  // To have seperate folder in Nebari server for downloading and executing files
+  if (nebari) {
+    common_nb_commands.unshift('%cd {cur_dir[0]}')
+    common_nb_commands.unshift(
+      'cur_dir = !mkdir pytorch-ignite-template-`date "+%Y%m%d-%H%M%S"` && cd $_ && echo $PWD'
+    )
+    execution_nb_commands.unshift('%cd {cur_dir[0]}')
+  }
```

Comment:
Is it possible to concat array in JS like in python:
```
a = [1, 2, 3] + a
```
?
If yes I would prefer that instead of unshifting.